.cci.jenkinsfile

```diff
@@ -6,6 +6,26 @@ cosaPod(buildroot: true) {
     // we don't need the qemu image to test coreos-installer; just the OSTree
     fcosBuild(make: true, skipKola: true, extraArgs: 'ostree')
 
+    stage("Check help") {
```

Comment:
I'd put this before `fcosBuild`; it's silly to build the full OS if we fail right after because of line length. That would involve calling `make` upfront, and then you can remove `make: true` from `fcosBuild()`.

(Even better would be doing it in parallel so that we get feedback on both branches in a single run.)